deploy/install/02-components/01-manager.yaml

```diff
@@ -80,3 +80,6 @@ spec:
         configMap:
           name: longhorn-default-setting
       serviceAccountName: longhorn-service-account
+  updateStrategy:
+    rollingUpdate:
```

Comment:
Should this be `OnDelete`? I am not sure `maxUnavailable` guarantees that all old pods are removed before the new pods starts